internal/dockercompose/wrapper.go

```diff
@@ -0,0 +1,135 @@
+package dockercompose
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/windmilleng/tilt/internal/logger"
+)
+
+type DockerComposeClient interface {
```

Comment:
i wonder if DockerComposeClient should be per docker-compose file, and then pathToConfig would just be a property of the client. (similar to how a db client would have the db address be a property of the client)

Reply:
yes I think that would be a good refactor. I'd rather leave this as is as it causes the fewest changes in the code but enables testing. Then we can make those changes much more confidently.

Reply:
ha yes i agree. leave a todo/ticket?